server/src/test/java/io/crate/expression/reference/doc/lucene/SourceParserTest.java

```diff
@@ -44,7 +45,7 @@ public class SourceParserTest extends ESTestCase {
 
     @Test
     public void test_extract_single_value_from_json_with_multiple_columns() throws Exception {
-        SourceParser sourceParser = new SourceParser();
+        SourceParser sourceParser = new SourceParser(Set.of());
```

Comment:
I chose not to expose a test-only ctor without args, since it's only used in this class, and it could be source of trouble in production code if used by accident.